validator/src/cli/thread_args.rs

```diff
@@ -116,6 +126,17 @@ trait ThreadArg {
     }
 }
 
+struct AccountsDbHashThreadsArg;
+impl ThreadArg for AccountsDbHashThreadsArg {
+    const NAME: &'static str = "accounts_db_hash_threads";
+    const LONG_NAME: &'static str = "accounts-db-hash-threads";
+    const HELP: &'static str = "Number of threads to use for background accounts hashing";
```

Comment:
nit: Is there any extra wording we can add here to distinguish this from the other pool (`thread_pool`) that will eventually be added. I think I've heard y'all use the terms foreground/background ? That other pool also does hashing (accounts delta hashing); I know these are private but some extra context probably useful for our team as well 😄